scripts/install_third_party_libs.py

```diff
@@ -211,6 +214,73 @@ def ensure_system_python_libraries_are_installed(package, version):
     install_backend_python_libs.pip_install_to_system(package, version)
 
 
+def rewrite_android_proto_files():
+    """Edit all android proto files as protobuf not
+    support config imports.
+    """
+    # Since there is no simple configuration for imports when using protobuf to
+    # generate Python files we need to manually fix the imports.
+    # See: https://github.com/protocolbuffers/protobuf/issues/1491
+    protobuf_dir = (
+        pathlib.Path(
+            os.path.join(
+                common.THIRD_PARTY_DIR,
+                'oppia-proto-api-introduce-proto-api-v1'))
+            .glob('**/*.proto'))
+    for p in protobuf_dir:
```

Comment:
Add a comment explaining roughly what this sequence of commands actually does. (You can perhaps show the before/after so that the overall combined effect of the changes is clear.)

Reply:
I had added comments at each level, does this sounds good?

```py
for p in protobuf_dir:
        if p.suffix == '.proto':
            # Remove package statement.
            # Example - 'package org.oppia.proto.v1.api'
            common.inplace_replace_file(
                p.absolute(),
                r'^package ([^\s]+)',
                r'')
            # Remove option statement.
            # Example - 'option java_package = "org.oppia.proto.v1.api";'
            common.inplace_replace_file(
                p.absolute(),
                r'^option java_.+',
                r'')
            # Update import statement.
            # Example - 'import "org/oppia/proto/v1/api/state.proto";'
            common.inplace_replace_file(
                p.absolute(),
                r'^import (?!\"google)([^\s]+)[\\/]([^\s]+)',
                r'import "\2')
            # Remove all subpackage directories.
            # Example - 'org.oppia.'
            common.inplace_replace_file(
                p.absolute(),
                r'org.oppia.*\.',
                r'')
```